scripts/demo.sh

```
@@ -1,128 +1,408 @@
+command -v tilt >/dev/null 2>&1 || { echo "tilt required. Install: brew install tilt"; exit 1; }
+echo -e "${GREEN}✓ All tools available${NC}\n"
+
+# Check if Tilt is running, start if needed
+echo -e "${YELLOW}Checking Tilt status...${NC}"
+if ! tilt get uisession >/dev/null 2>&1; then
+    echo -e "${YELLOW}Tilt not running. Starting Tilt in background...${NC}"
+    echo -e "${YELLOW}This may take 2-3 minutes for initial startup.${NC}"
+
+    # Start Tilt in background
+    tilt up &
+    TILT_PID=$!
```

Shellcheck (0.11.0)

[warning] TILT_PID appears unused. Verify use (or export if used externally).

(SC2034)